.readthedocs.yaml

```
@@ -0,0 +1,24 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# RTD API version
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: >-
+      3.11
+  # Build with make coredocs
+  commands:
+    - python -m venv "${READTHEDOCS_VIRTUALENV_PATH}"
+    - >-
+      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -m pip install --exists-action=w -r tests/requirements.in -c tests/requirements.txt
+    - >-
+      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python docs/bin/clone-core.py
+    - >-
+      make coredocs
+      -C docs/docsite
+      PYTHON="${READTHEDOCS_VIRTUALENV_PATH}"/bin/python
+    - mkdir -pv "${READTHEDOCS_OUTPUT}"/html
+    - mv -v docs/docsite/_build/html "${READTHEDOCS_OUTPUT}"/
```

Looks like these lines can be replaced with just
```suggestion
      BUILDDIR="${READTHEDOCS_OUTPUT}"
      PYTHON="${READTHEDOCS_VIRTUALENV_PATH}"/bin/python
```